src/components/AssetManagementPanel/AssetManagementForm/FormVariants/SurrenderForm/SurrenderForm.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { SurrenderForm } from "./SurrenderForm";
+import { AssetManagementActions } from "../../../AssetManagementContainer";
+
+describe("Surrender", () => {
+  xit("should display the non-editable beneficiary & holder when the app is in Surrender state", () => {});
```

Comment:
Can test this by checking for the text directly le

Reply:
added the test to check for text directly